views/marts/product/dim_daily_server_info.view.lkml

```diff
@@ -0,0 +1,173 @@
+    type: string
+    sql: ${TABLE}.database_type ;;
+    label: "Database Type"
+    description: "The type of the database"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: database_version {
+    type: string
+    sql: ${TABLE}.database_version ;;
+    label: "Database Version"
+    description: "The version of the database"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: installation_id {
+    type: string
+    sql: ${TABLE}.installation_id ;;
+    label: "Installation ID"
+    description: "The server's installation ID (if any)"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: installation_type {
+    type: string
+    sql: ${TABLE}.installation_type ;;
+    label: "Installation type"
+    description: "The server's installation type (if any)"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: is_cloud {
+    type: yesno
+    sql: ${TABLE}.is_cloud ;;
+    label: "Is Cloud?"
+    description: "Whether this is a cloud server"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: is_enterprise_ready {
+    type: string
+    sql: ${TABLE}.is_enterprise_ready ;;
+    label: "Is Enterprise Ready?"
+    description: "Whether this is an enterprise ready server"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: operating_system {
+    type: string
+    sql: ${TABLE}.operating_system ;;
+    label: "Operating System"
+    description: "The OS the server is installed on"
+    view_label: "- Server Metadata"
+  }
+
+  dimension: server_ip {
+    type: string
+    sql: ${TABLE}.server_ip ;;
+    label: "IP Address"
+    description: "The server's IP address"
+    view_label: "- Server Metadata"
+  }
+
+  ###
+  ### Version Information
+  ###
+
+  dimension: version_full {
+    type: string
+    sql: ${TABLE}.version_full ;;
+    label: "Server Version"
+    view_label: "- Server Version"
+  }
+
+  dimension: version_major {
+    type: string
+    sql: ${TABLE}.version_major ;;
+    label: "Major Version"
+    view_label: "- Server Version"
+  }
+
+  dimension: version_minor {
+    type: string
+    sql: ${TABLE}.version_minor ;;
+    label: "Minor Version"
+    view_label: "- Server Version"
+  }
+
+  dimension: version_patch {
+    type: string
+    sql: ${TABLE}.version_patch ;;
+    label: "Patch Version"
+    view_label: "- Server Version"
+  }
+
+  dimension: count_reported_versions {
+    type: number
+    sql: ${TABLE}.count_reported_versions ;;
+    label: "Number of reported versions"
+    view_label: "- Server Version"
+  }
+
+  ###
+  ### Data Source info
+  ###
+
+  dimension: has_diagnostics_data {
+    type: yesno
+    sql: ${TABLE}.has_diagnostics_data ;;
+    label: "Has diagnostics data?"
+    description: " Whether the server queried the security update endpoint at the current date"
+    view_label: "- Data Source Info"
+  }
+
+  dimension: has_legacy_telemetry_data {
+    type: yesno
+    sql: ${TABLE}.has_legacy_telemetry_data ;;
+    label: "Has legacy telemetry"
```

Comment:
Does the "?" need to be part of the label for a `yesno` type or is on a case by case?

Reply:
Most likely a typo. We can decide if we like it or not and adjust. I'll add it for now just to be consistent, and we can revise later.